docs/faq.rst

```diff
@@ -59,3 +59,60 @@ This error can occure within requests' ``session.py`` when called by
 the destructor (``__del__``) of browser. The solution is to
 call :func:`~mechanicalsoup.Browser.close` before the end of life of
 the object.
+
+How do I get debug information/logs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To unerstand what's going on while running a script, you have two
```

Comment:
Typo in "understand" here.

Reply:
Oups, and it was not even the only typo. A spell checker does help ;-).